changelog.d/saf-897.fixed

```diff
@@ -0,0 +1,3 @@
+Improved removal of pre-existing findings in inter-file diff scan: We've
```

Comment:
I'd describe what was happening, "when renaming or adding new files, inter-file diff scans could report pre-existing findings". More than improving it, we've fixed the problem IMO.

Reply:
done.